src/App.js

```diff
@@ -0,0 +1,17 @@
+import "./App.css";
+import { BiSearchAlt } from "react-icons/bi";
+import LandingPage from "./Pages/LandingPage";
+import { Routes, Route } from "react-router-dom";
+import ExplorePage from "./Pages/ExplorePage";
```

Comment:
you can import `LandingPage` and `ExplorePage` components directly together from the `index.js` file which will optimize your code better